File: Fast-&-Lethal/seth/Seth_Champion.py
class SethChampion:
    def __init__(self):
        self.name = "Seth"
        self.title = "O Ceifador Sombrio"
        self.health = 1000
        self.essence_of_void = 0
        self.mark_of_death_active = False

    def regenerate_essence(self):
        if self.essence_of_void < 100:
            self.essence_of_void += 10
            print(f"Essência do Void: {self.essence_of_void}")

    def ravenous_dagger(self):
        print(f"{self.name} usa Ravenous Dagger! Dano verdadeiro + sangramento por 3s.")
        self.apply_mark_of_death()

    def swift_bow(self, enemies_hit):
        print(f"{self.name} usa Swift Bow! Causa lentidão e roubo de vida.")
        if enemies_hit >= 3:
            print("Seth ganha um escudo temporário.")
        self.apply_mark_of_death()

    def vertical_slash(self, target_bleeding):
        print(f"{self.name} usa Vertical Slash! Dano verdadeiro + cegueira + sangramento.")
        if target_bleeding:
            print("Efeitos prolongados por 1 segundo!")
        self.apply_mark_of_death()

    def slaughter(self, can_execute, team_ace):
        if can_execute:
            print(f"{self.name} usa Slaughter! Executando inimigo(s).")
            if team_ace:
                print("Buff de dano verdadeiro concedido aos aliados por 6s!")
        else:
            print("Slaughter não pode ser ativado: Marca da Morte não está ativa.")

    def apply_mark_of_death(self):
        self.mark_of_death_active = True
        print("Marca da Morte aplicada ao alvo.")


if __name__ == "__main__":
    seth = SethChampion()
    seth.regenerate_essence()
    seth.ravenous_dagger()
    seth.swift_bow(3)
    seth.vertical_slash(True)
    seth.slaughter(True, True)

File: Fast-&-Lethal/seth/Seth_Champion.java
public class SethChampion {
    private final String name = "Seth";
    private final String title = "O Ceifador Sombrio";
    private int health;
    private int essenceOfVoid;
    private boolean markOfDeathActive;

    public SethChampion() {
        this.health = 1000;
        this.essenceOfVoid = 0;
        this.markOfDeathActive = false;
    }

    public void regenerateEssence() {
        if (essenceOfVoid < 100) {
            essenceOfVoid += 10;
            System.out.println("Essência do Void: " + essenceOfVoid);
        }
    }

    public void ravenousDagger() {
        System.out.println(name + " usa Ravenous Dagger! Dano verdadeiro + sangramento por 3s.");
        applyMarkOfDeath();
    }

    public void swiftBow(int enemiesHit) {
        System.out.println(name + " usa Swift Bow! Causa lentidão e roubo de vida.");
        if (enemiesHit >= 3) {
            System.out.println("Seth ganha um escudo temporário.");
        }
        applyMarkOfDeath();
    }

    public void verticalSlash(boolean targetBleeding) {
        System.out.println(name + " usa Vertical Slash! Dano verdadeiro + cegueira + sangramento.");
        if (targetBleeding) {
            System.out.println("Efeitos prolongados por 1 segundo!");
        }
        applyMarkOfDeath();
    }

    public void slaughter(boolean canExecute, boolean teamAce) {
        if (canExecute) {
            System.out.println(name + " usa Slaughter! Executando inimigo(s).");
            if (teamAce) {
                System.out.println("Buff de dano verdadeiro concedido aos aliados por 6s!");
            }
        } else {
            System.out.println("Slaughter não pode ser ativado: Marca da Morte não está ativa.");
        }
    }

    private void applyMarkOfDeath() {
        this.markOfDeathActive = true;
        System.out.println("Marca da Morte aplicada ao alvo.");
    }

    public static void main(String[] args) {
        SethChampion seth = new SethChampion();
        seth.regenerateEssence();
        seth.ravenousDagger();
        seth.swiftBow(3);
        seth.verticalSlash(true);
        seth.slaughter(true, true);
    }
}